### PR TITLE
[StyleCop] Fix warnings in Bot.Connetor.Test project

### PR DESCRIPTION
**Pull request by ceciliaavila**
- Reorder using directives
- Remove empty blank lines
- Add trailing comma in multi-line initializers
- Remove trailing white spaces
- Update protected const fields to be protected property with private set.
- Update private const field to be private read-only to avoid unreachable code in BaseTest class.

### CODE REVIEW

File: tests/Microsoft.Bot.Connector.Tests/BaseTest.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@ namespace Connector.Tests
 {
     using System;
     using System.Collections.Generic;
+    using System.Diagnostics.CodeAnalysis;
```
Comment (munozemilio) [inline tests/Microsoft.Bot.Connector.Tests/BaseTest.cs]: Is this used?

File: tests/Microsoft.Bot.Connector.Tests/BotAccessTokenStub.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Rest;
+
+namespace Connector.Tests
```
Comment (munozemilio) [inline tests/Microsoft.Bot.Connector.Tests/BotAccessTokenStub.cs]: Can we add full namespace?

File: tests/Microsoft.Bot.Connector.Tests/BaseTest.cs (hunk 1 above)
Comment (GasparAcevedoZainSouthworks) [reply tests/Microsoft.Bot.Connector.Tests/BaseTest.cs]: Feedback applied

File: tests/Microsoft.Bot.Connector.Tests/BotAccessTokenStub.cs (hunk 2 above)
Comment (GasparAcevedoZainSouthworks) [reply tests/Microsoft.Bot.Connector.Tests/BotAccessTokenStub.cs]: Feedback applied

Comment (GasparAcevedoZainSouthworks) [reply tests/Microsoft.Bot.Connector.Tests/BotAccessTokenStub.cs]: Feedback Applied

**Pull request merged**